### DISCUSSION TITLE
Support Bookmark Export or API

### DISCUSSION THREAD

**Discussion by DanHugoDanHugo** [dailydotdev/daily#640] [Feature Requests]
**Minimum**
There should be a straightforward Export UI that enables export of all bookmarks to a standard bookmarks.html file suitable for import into your typical web browser.

**Nice to Have**
How about exposing a basic API to export a range of APIs (eg date range, for local sync) or with other filtering?

**All in**
How about a real Bookmark Manager UI (this has been suggested elsewhere in these discussions) that enables import perhaps, definitely export, based on range, keyword, etc, that would essentially make use of the Nice to Have API otherwise exposed?

**Super Project**
Export of Bookmarks should be a part of a larger Export User Data feature which most if not all public platforms support nowadays to enable a user to either keep a copy of their data locally, or to take their data and depart the platform.

**Comment by idoshamun** [comment]
Hi @DanHugoDanHugo,

Thanks a lot for the suggestion.
I agree that import/export is a must for interoperability with other tools and that our bookmarks feature needs more advanced options, as you and others mentioned.
Unfortunately, we are still a small team, and this is out of our scope. I hope we will be able to address this concern in the future.

**Reply by DanHugoDanHugo** [reply]
I do appreciate the challenges of a small team, but…
Time spent screwing up the UI could have been spent on bookmark export.

Good luck.

**Reply by idoshamun** [reply]
Thanks for the direct feedback. We now understand it's a controversial change (originated from users' feedback btw). We will think what's the best way to make everyone satisfied. I appreciate your patience, don't lose hope in us. 😬

**Comment by capJavert** [comment]
**Update: daily.dev now has a public API** 🎉

This one's been asked for here for a long time, so: it's live, and it's
available to everyone. Generate a personal access token at
[daily.dev/settings/api](https://daily.dev/settings/api) and start calling it.

What's in v1:

- **Feeds** — your personalized For You feed, trending posts, filtering by tag or source
- **Posts** — full post details, AI summaries and comment threads
- **Search** — posts, tags and sources
- **Bookmarks** — read and manage your saved content
- **Notifications** — your activity feed and unread counts
- **Profile and reading streak** — your experience, tech stack and streak

📣 The announcement: https://daily.dev/posts/public-api-access-is-now-open-for-everyone-ljj8kh9sa
📚 Docs: https://docs.daily.dev/public-api/
